### ISSUE TITLE
Raj092306

### ISSUE THREAD

**Issue by Julief8** [rmercer33/christmind.info#28]
On this one you should probably look at your version of the md file first.  I made a number of small corrections and this are one below and I skipped one bullseye as discussed below...

[092306.md.zip](https://github.com/rmercer33/christmind.info/files/1118253/092306.md.zip)

A sentence "So would we unite with them." was eliminated and the bullseye was skipped because it was not said by Raj.  It is the same sentence that ended the prior paragraph and since both paragraphs were quotes from ACIM I also fixed the wrap but left them as separate paragraphs so the timing is correct.  Hope this is understandable.  If not let me know.  Cinta,  Julie

**Comment by rmercer33** [comment]
Yes, I see what you're saying. In this case we should not delete 'So would we...' because it will effect results of a search. For example, if a search finds a match in paragraphs following the deleted line the match will have the wrong paragraph number - it will be off by one.

So, in this case we should put an {: .omit} in column one following that line.  That will cause the paragraph to not be counted and the audio should match correctly.

I will change that part back and see if it works as expected.

**Comment by rmercer33** [comment]
I changed it and it "looks like" its working as expected - BUT, there is no way to know exactly unless I don't fast forward - because of the 'time effect' of seeking.

That needs to be corrected.

Need to come back to this after #3 is fixed
